boot/boot_serial/src/boot_serial.c

```diff
@@ -64,7 +64,7 @@
 BOOT_LOG_MODULE_DECLARE(mcuboot);
 
 #define BOOT_SERIAL_INPUT_MAX   512
-#define BOOT_SERIAL_OUT_MAX     128
```

Comment:
let's do this conditional. Can we make this value dependent on image-pairs number?

Reply:
Fixed